service.go

```diff
@@ -12,19 +13,23 @@ type ServiceId struct {
 }
 
 type Service struct {
-	Description string `json:"description,omitempty"`
-	Framework   string `json:"framework,omitempty"`
-	HtmlURL     string `json:"htmlUrl"`
+	ApiDocumentPath string `json:apiDocumentPath,omitempty`
```

Comment:
Should this have double quotes like the others?

```suggestion
	ApiDocumentPath string `json:"apiDocumentPath,omitempty"`
```

Reply:
Great Catch!